test_fms/data_override/test_data_override2.sh

```diff
@@ -23,21 +23,29 @@
 # execute tests in the test_fms/data_override directory.
 
 # Ed Hartnett 11/26/19
+# Uriel Ramirez 07/22/20
 
 # Set common test settings.
 . ../test_common.sh
 
+# Run the ongrid test case:
+touch input.nml
+printf '"OCN", "runoff", "runoff", "./INPUT/runoff.daitren.clim.1440x1080.v20180328.nc", "none" ,  1.0' | cat > data_table
+[ ! -d "INPUT" ] && mkdir -p "INPUT"
+run_test test_data_override_ongrid 6
+rm -rf "INPUT"
+
 # Copy to builddir and rename data files for tests.
-cp $top_srcdir/test_fms/data_override/data_table_base data_table
-cp $top_srcdir/test_fms/data_override/diag_table_base diag_table
+#cp $top_srcdir/test_fms/data_override/data_table_base data_table
+#cp $top_srcdir/test_fms/data_override/diag_table_base diag_table
 #cp -r $top_srcdir/test_fms/data_override/INPUT $top_builddir/test_fms/data_override/INPUT
 
 # Both tests are skipped in the bats file, so commented out here.
-tnum=$( printf "%2.2d" 1 )
-sed "s/<test_num>/${tnum}/"  $top_srcdir/test_fms/data_override/input_base.nml > input.nml
-run_test test_data_override 2 skip
+#tnum=$( printf "%2.2d" 1 )
+#sed "s/<test_num>/${tnum}/"  $top_srcdir/test_fms/data_override/input_base.nml > input.nml
+#run_test test_data_override 2 skip
 
-tnum=$( printf "%2.2d" 2 )
-sed "s/<test_num>/${tnum}/"  $top_srcdir/test_fms/data_override/input_base.nml > input.nml
-run_test test_data_override 2 skip
+#tnum=$( printf "%2.2d" 2 )
+#sed "s/<test_num>/${tnum}/"  $top_srcdir/test_fms/data_override/input_base.nml > input.nml
+#run_test test_data_override 2 skip
```

Comment:
Is this test obsolete?  It's completely commented out.  Why not just delete?

Reply:
It is commented out because it doesn't run and it doesn't run because there are input netcdf files missing. 
It tests something different than the test I added.

Reply:
OK, so it's there for the future then?

Reply:
yes